### DISCUSSION TITLE
Lesson 6: Reverted: Exception While Processing Transaction Using ganache-local

### DISCUSSION THREAD

**Discussion by hai-yr** [smartcontractkit/full-blockchain-solidity-course-py#778] [Q&A] [answered]
When I run `brownie test` using the ganache-cli, both the tests pass, but when I run `brownie test --network ganache-local` I get 
"ValueError: Gas estimation failed: 'execution reverted: VM Exception while processing transaction: revert'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually." Is this an issue with the ganache GUI or does it require me to change the code to get this to work. Thanks!

**Comment by cromewar** [comment, marked as answer]
Hello @hai-yr, this error already has been reported (not remember which thread tough) the solution was to enter to ganache UI configuration and increase the gas limit for the instance you are running.
